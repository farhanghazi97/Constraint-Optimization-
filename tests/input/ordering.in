# test that output is in order that meeting were defined
meeting, m3
meeting, m2
meeting, m1
domain, m1, tue, hard
domain, m1, 10am, hard
domain, m2, fri, hard
domain, m2, 2pm, hard
domain, m3, mon, hard
domain, m3, 12pm, hard
