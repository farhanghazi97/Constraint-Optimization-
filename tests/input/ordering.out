m3:mon 12pm
m2:fri 2pm
m1:tue 10am
cost:0
